libraries/ts/src/sbv2.ts

```diff
@@ -1148,16 +1160,16 @@ export class AggregatorAccount {
       params.authority ?? this.keypair ?? programWallet(this.program);
     return program.methods
       .aggregatorSetConfig({
-        name: findKeyInObject(params, "name"),
-        metadata: findKeyInObject(params, "metadata"),
-        batchSize: findKeyInObject(params, "batchSize"),
-        minOracleResults: findKeyInObject(params, "minOracleResults"),
-        minUpdateDelaySeconds: findKeyInObject(params, "minUpdateDelaySeconds"),
-        minJobResults: findKeyInObject(params, "minJobResults"),
-        forceReportPeriod: findKeyInObject(params, "forceReportPeriod"),
-        varianceThreshold: findKeyInObject(params, "varianceThreshold", (o) =>
-          SwitchboardDecimal.fromBig(new Big(o))
-        ),
+        name: params.name ?? null,
+        metadata: params.metadata ?? null,
+        batchSize: params.batchSize ?? null,
+        minOracleResults: params.minOracleResults ?? null,
+        minUpdateDelaySeconds: params.minUpdateDelaySeconds ?? null,
+        minJobResults: params.minJobResults ?? null,
+        forceReportPeriod: params.forceReportPeriod ?? null,
+        varianceThreshold: lodash.isFinite(params.varianceThreshold)
```

Comment:
this shouldnt be a boolean

Reply:
Its not - I check whether the parameter is a finite number - and if so, need to transform it to a `SwitchboardDecimal`. If the number isnt a finite number, `null` is used.
```typescript
varianceThreshold: lodash.isFinite(params.varianceThreshold)
  ? SwitchboardDecimal.fromBig(new Big(params.varianceThreshold))
  : null,
```